products/amazon-neptune.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Amazon-Neptune
+category: db
+tags: amazon
+permalink: https://docs.aws.amazon.com/neptune/latest/userguide/engine-releases.html
+auto:
+-   custom: true
+releases:
+-   releaseCycle: "1.2"
+    releaseDate: 2022-07-21
+    eol: 2025-10-30
+    latest: "1.2.1.0"
+    latestReleaseDate: 2023-03-08
+    
+-   releaseCycle: "1.1"
+    releaseDate: 2021-11-19
+    eol: 2024-10-31
+    latest: "1.1.1.0"
+    latestReleaseDate: 2022-04-19
+
+-   releaseCycle: "1.0"
+    releaseDate: 2019-07-02
+    eol: 2023-01-30
+    latest: "1.0.5.1"
+    latestReleaseDate: 2021-10-01
+    
+-   releaseCycle: "1.0"
+    releaseDate: 2021-07-27
+    eol: 2023-01-30
+    latest: "1.0.5.1"
+    latestReleaseDate: 2021-10-01  
+---
+
+> [Amazon Neptune](https://docs.aws.amazon.com/neptune/index.html) is a fast, reliable,
+>  fully managed graph database service that makes it easy to build and run applications
+>  that work with highly connected datasets. It supports multiple property-graph
+>  query languages - Apache TinkerPop Gremlin, openCypher, and SPARQL. 
+>  Neptune powers graph use cases such as recommendation engines,
+>  fraud detection, knowledge graphs, drug discovery, and network security.
+
+Neptune engine versions almost always reach their end-of-life at the end of a calendar quarter. 
```

Comment:
I've edited things slightly, but a short note on style: We prefer using [neutral third-person voice](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#tone-and-text), which means avoiding words like "We/Our" (we aren't speaking on behalf of AWS), or "you" (instead preferring neutral terms like "users/customers", or avoiding them altogether).

Could you take a shot at fixing this? Happy to help if you have any questions.

Reply:
Made some changes, closing as resolved